runtime

```diff
@@ -25,10 +25,25 @@ case $1 in
         popd
         ;;
 
+    clean)
+        pushd shm_wrapper && make clean && popd
+        pushd net_handler && make clean && popd
+        pushd executor    && make clean && popd
+        pushd dev_handler && make clean && popd
+        pushd tests       && make clean && popd
+        rm -f /tmp/log-fifo
+        rm -f /tmp/challenge.sock
+        rm -f /tmp/ttyACM*
+        ;;
+    
```

Comment:
Out of curiosity, why the reorder? It seems the only thing that changed right here was the fact you moved it higher in the code

Reply:
Alphabetical order I think? @DanielMolina24

Reply:
Oh ok I see!

Reply:
Oh is it in alphabetical order? :P 

I just reordered it so the similar ones were closer to together (ex: build and clean; run and stop)